app/views/foreman_tasks/recurring_logics/index.html.erb

```diff
@@ -5,6 +5,13 @@
 <%= alert(:class => 'alert-info', :id => 'multiple-alert', :close => false, :header => '', :text => @errors) %>
 <% end %>
 
+<% if authorized_for(:permission => :edit_recurring_logics, :auth_object => @recurring_logics) %>
+  <% title_actions link_to(_('Clear Cancelled'),
+              clear_cancelled_foreman_tasks_recurring_logics_path,
+              class:  ['btn', 'btn-sm', 'btn-primary'],
```

Comment:
Shouldn't the button be red to denote it is a possibly destructive action?

Reply:
Yeah you're right. I also added a confirm dialog